samples/java/appgengine-java11/simple-server/README.md

```diff
@@ -0,0 +1,87 @@
+# Using the Java Agent on a Simple One File Server App in Google App Engine Standard Java 11
+
+NOTE: This example was copied from
+[appengine-java11/custom-entrypoint](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/appengine-java11/custom-entrypoint)
+and modified for Snapshot Debugger Java agent use.
+
+
+## Setup
+See [Prerequisites](../README.md#Prerequisites).
+
+## Compile the Source With Full Debug Information Enabled
+
+The `-g` option here is what enables full debug information. Without it
+information on method arguments and local variables would not be available. The
+`-source` and `-target` options are to ensure it will run with the Java 11
+runtime.
+
+```
+javac -source 11 -target 11 -g Main.java
+```
+
+## Install the Snapshot Debugger Java Agent
+
+```
+mkdir cdbg
+wget -qO- https://github.com/GoogleCloudPlatform/cloud-debug-java/releases/latest/download/cdbg_java_agent_gce.tar.gz | tar xvz -C cdbg
+```
+
+## Deploy to App Engine Standard
+
+Examine the app.yaml contents, which provides a custom entry point that
+specifies the `-agentpath` java option to load the agent:
+
+https://github.com/GoogleCloudPlatform/snapshot-debugger/blob/cbe7516dc06d4f8c591b0239fa5cea078787f7c5/samples/java/appgengine-java11/simple-server/app.yaml#L15-L16)
+
+Deploy the app with the following:
+
+```
+gcloud app deploy
+```
+
+Make note of the following output entries, which should resemble the following:
```

Comment:
Just a thought: Setting `service: sample-java11-simple-server` in the app.yaml would make this easier.

Reply:
Good idea, thanks!